packages/x-date-pickers/src/DesktopTimePicker/tests/DesktopTimePicker.test.tsx

```diff
@@ -82,9 +82,8 @@ describe('<DesktopTimePicker />', () => {
       await user.click(screen.getByRole('option', { name: '09:00 AM' }));
       expect(onChange.callCount).to.equal(1);
       expect(onChange.lastCall.args[0]).toEqualDateTime(new Date(2018, 0, 1, 9, 0));
-      expect(onAccept.callCount).to.equal(1);
-      expect(onAccept.lastCall.args[0]).toEqualDateTime(new Date(2018, 0, 1, 9, 0));
-      expect(onClose.callCount).to.equal(1);
+      expect(onAccept.callCount).to.equal(0); // onAccept will only called when onCloseOnSelect is true (false by default for pickers with time view)
+      expect(onClose.callCount).to.equal(0); // onClose will only called when onCloseOnSelect is true (false by default for pickers with time view)
     });
 
     it('should call "onAccept", "onChange", and "onClose" when selecting all section', async () => {
```

Comment:
I think that we can remove this test, given almost exactly the same assertion in `testPickersOpenCloseLifecycle`.